src/slab_data.c

```diff
@@ -255,7 +255,8 @@ TbBool slab_kind_is_animated(SlabKind slbkind)
 {
     if (slab_kind_is_door(slbkind))
         return true;
-    if ((slbkind == SlbT_GUARDPOST) || (slbkind == SlbT_BRIDGE) || (slbkind == SlbT_GEMS))
+    // if ((slbkind == SlbT_GUARDPOST) || (slbkind == SlbT_BRIDGE) || (slbkind == SlbT_GEMS))
+        if (slbkind >= 50)
```

Comment:
Are we really sure all slabs over 50 are animated?
And even if we are, it would still be better to refer to the enum value of 50, so that if slabs are added in between the function keeps working as expected.

Reply:
It seems to me that all slabs over 50 are animated.